docs/dev/plugins.rst

```diff
@@ -211,6 +211,22 @@ The events currently available are:
 * *import_begin*: called just before a ``beet import`` session starts up.
   Parameter: ``session``.
 
+* *trackinfo_received*: called after meta data for a track item has been fetched
+  from disparate sources, such as MusicBrainz. Gives a developer the option to
+  intercept the fetched TrackInfo object. Can be used to modify tags on a ``beet
+  import`` operation or during later adjustments, such as ``mbsync``. Can be
+  slow, as event is fired for any fetched possible match *before* user or
+  autotagger selection was made.
+  Parameter: ``info``.
+
+* *albuminfo_received*: called after meta data for an album item has been
```

Comment:
Perhaps this bullet could use a little less duplicated text, and just say "like `trackinfo_received`, it indicates new medata, but supplies an `AlbumInfo` object instead of a `TrackInfo`".